src/bitcoin/bitcoinCoreRpc.ts

```diff
@@ -68,7 +68,8 @@ export class BitcoinCoreRpc implements BitcoinBlockchain {
       username,
       password,
       host,
-      port
+      port,
+      timeout: 60000
```

Comment:
Can we have a const instead of a magic number please (with an comment on how the value was chosen).  Thanks

Reply:
I think inline is fine as this value is only used in here and it should not change. 
Also, there is no scientific reason behind this value besides: _just increase it, deploy it_ and see if it reduces the timeouts :D